src/views/project/institutions/CreateView.vue

```diff
@@ -78,6 +98,15 @@ async function searchInstitutions() {
             {{ institution.name }}
           </option>
         </select>
+
+        <button
+          type="button"
+          class="btn btn-m btn-outline-primary"
+          @click="buildInstitution()"
+        >
+          <i class="fa fa-plus"></i>
+          <span>Add Institution</span>
+        </button>
```

Comment:
Having a button for a text input that _sometimes_ is needed would be very confusing.

What about this? Let's have a textbox with a button. The user will input into the textbox for institutions which will search for pre-existing institutions:
1. if the user selects the institution, they must click "Save" to add that institution to the project.
2. if the user creates a new institution, they must also click "Save" which will create the institution and add it to the project.

From the user's point of view, they will type and institutions and click "Save".